src/main/java/uk/gov/hmcts/darts/common/entity/base/CreatedBaseEntity.java

```diff
@@ -24,6 +25,7 @@ public class CreatedBaseEntity {
     @NotAudited
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "created_by")
+    @CreatedBy
```

Comment:
It seems we have `CreatedBaseEntity` and a `MandatoryCreatedBaseEntity`, I presume which one to use depends on whether the entity has a `NOT NULL` constraint on the respective DB field.

So for the 16 usages of `CreatedBaseEntity` are we sure we always want `createdBy` to be automatically set?

Reply:
I must have missed these so will stick the same annotation on these ones :)